### DISCUSSION TITLE
VScode Authentication failure!

### DISCUSSION THREAD

**Discussion by CMFerreiraEco** [community/community#6709] [Projects and Issues] [answered]
I just received an email with the invitation to be part of the technical preview and despite being logged to my git in the vscode the copilot keeps saying that I do not have access! 

Cheers in advance!!!!

**Comment by davecheney** [comment, marked as answer]
Please check https://github.com/github/feedback/discussions/6637

**Reply by CMFerreiraEco** [reply]
Thank you very much mate, problem solved! All set and ready to fly now! 
cheers,

**Comment by Naifalhazmy** [comment]
[](url)
